ui-admin/src/study/participants/participantList/ParticipantListTableGroupedByFamily.tsx

```diff
@@ -0,0 +1,207 @@
+import React, {
+  useMemo,
+  useState
+} from 'react'
+import Api, { EnrolleeSearchExpressionResult } from 'api/api'
+import { StudyEnvContextT } from '../../StudyEnvironmentRouter'
+import {
+  ColumnDef,
+  getCoreRowModel,
+  getExpandedRowModel,
+  getFilteredRowModel,
+  getPaginationRowModel,
+  getSortedRowModel,
+  Row,
+  SortingState,
+  useReactTable
+} from '@tanstack/react-table'
+import {
+  basicTableLayout,
+  renderEmptyMessage,
+  useRoutableTablePaging
+} from 'util/tableUtils'
+import {
+  Family,
+  instantToDefaultString
+} from '@juniper/ui-core'
+import TableClientPagination from 'util/TablePagination'
+import ParticipantListTable from 'study/participants/participantList/ParticipantListTable'
+import { getFamilyNames } from 'util/familyUtils'
+import { NavLink } from 'react-router-dom'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import {
+  faChevronDown,
+  faChevronUp
+} from '@fortawesome/free-solid-svg-icons'
+import { useLoadingEffect } from 'api/api-utils'
+import LoadingSpinner from 'util/LoadingSpinner'
+import { EnrolleeLink } from 'study/participants/enrolleeView/EnrolleeLink'
+
+type FamilyWithSearchResults = Partial<Family> & { searchResults: EnrolleeSearchExpressionResult[] }
+
+/**
+ * Shows a table of participants grouped by their family.
+ */
+function ParticipantListTableGroupedByFamily({
+  participantList,
+  studyEnvContext
+}: {
+  participantList: EnrolleeSearchExpressionResult[],
+  studyEnvContext: StudyEnvContextT
+}) {
+  const { paginationState, preferredNumRowsKey } = useRoutableTablePaging('participantList')
+
+  const [sorting, setSorting] = useState<SortingState>([{ 'id': 'createdAt', 'desc': true }])
+  const [families, setFamilies] = useState<Family[]>([])
+
+  const { isLoading } = useLoadingEffect(async () => {
+    const loadedFamilies = await Api.getAllFamilies(
+      studyEnvContext.portal.shortcode,
+      studyEnvContext.study.shortcode,
+      studyEnvContext.currentEnv.environmentName)
+    setFamilies(loadedFamilies)
+  }, [])
+
+  const columns = useMemo<ColumnDef<FamilyWithSearchResults>[]>(() => [{
+    header: '',
+    accessorKey: 'expanded',
+    enableColumnFilter: false,
+    enableColumnSort: false,
+    cell: ({ row }) => {
+      return <>
+        <button
+          className="btn btn-link m-0 p-0"
+          onClick={() => row.toggleExpanded()}>
+          {row.getIsExpanded()
```

Comment:
switch these to match the expand/collapse actions in the sidebar

Reply:
🤔 Hmm, they match on my machine; is it possible you didn't get the latest commits?

<img width="400" alt="image" src="https://github.com/user-attachments/assets/8acf9df8-62b1-4213-87b5-ac549e4839c9">